frontend/public/components/routes.tsx

```diff
@@ -353,69 +324,82 @@ const RouteTargetRow: React.FC<RouteTargetRowProps> = ({ route, target }) => (
   </tr>
 );
 
-const CustomRouteHelp: React.FC<CustomRouteHelpProps> = ({ host, routerCanonicalHostname }) => (
-  <Popover
-    headerContent={<>Custom Route</>}
-    bodyContent={
-      <div>
-        <p>
-          To use a custom route, you must update your DNS provider by creating a canonical name
-          (CNAME) record. Your CNAME record should point to your custom domain{' '}
-          <strong>{host}</strong>, to the OpenShift canonical router hostname,{' '}
-          <strong>{routerCanonicalHostname}</strong>, as the alias.
-        </p>
-      </div>
-    }
-  >
-    <Button className="pf-m-link--align-left" type="button" variant="link">
-      <QuestionCircleIcon /> Do you need to set up custom DNS?
-    </Button>
-  </Popover>
-);
+const CustomRouteHelp: React.FC<CustomRouteHelpProps> = ({ host, routerCanonicalHostname }) => {
+  const { t } = useTranslation();
+  return (
+    <Popover
+      headerContent={<>{t('network-route~Custom route')}</>}
+      bodyContent={
+        <div>
+          <p>
+            <Trans i18nKey="network-route~custom_route">
+              To use a custom route, you must update your DNS provider by creating a canonical name
+              (CNAME) record. Your CNAME record should point to your custom domain{' '}
+              <strong>{{ host }}</strong>, to the OpenShift canonical router hostname,{' '}
+              <strong>{{ routerCanonicalHostname }}</strong>, as the alias.
+            </Trans>
```

Comment:
Worried about keys here too. If you do something like `<Trans t={t} ns="network-route">` you can use the text as the key.

Reply:
Changed to use `<Trans />` element as suggested, thanks!